docs/developer/advanced/sharing-saved-objects.asciidoc

```diff
@@ -334,12 +339,56 @@ image::images/sharing-saved-objects-step-5.png["Sharing Saved Objects ESO migrat
 
 NOTE: Reminder, don't forget to add unit tests and integration tests!
 
+[[sharing-saved-objects-phase-2]]
+=== Phase 2 Developer Flowchart
+
+This section covers switching a share-capable object type into a shareable one _or_ creating a new shareable saved object type. Refer to
+this flowchart:
+
+image::images/sharing-saved-objects-phase-2-dev-flowchart.png["Sharing Saved Objects phase 2 - developer flowchart"]
+
 [[sharing-saved-objects-step-6]]
-=== Step 6
+==== Step 6
+
+> *Update your _server-side code_ to mark these objects as "shareable"*
+
+When you register your object, you need to set the proper `namespaceType`. If you have an existing object type that is "share-capable", you
+can simply change it:
+
+image::images/sharing-saved-objects-step-6.png["Sharing Saved Objects registration (shareable)"]
+
+[TIP]
+====
+You can also mark your object as
+https://github.com/elastic/kibana/blob/{branch}/docs/development/core/server/kibana-plugin-core-server.savedobjectstypemanagementdefinition.md[importable and exportable]:
+
+```json
+management: { isImportableAndExportable: true },
+```
+
+If you do this, then your objects will be visible in the <<managing-saved-objects,Saved Objects Management page>>, where users can assign
+them to multiple spaces.
+====
```

Comment:
I'm not sold on this tip yet. It gives me the impression that the only way to share objects is via SOM, and I don't think that's the message we want to send.

What if we re-worded `Step 8` from:
> *Update your _client-side code_ to add reusable components*

to something like:
> *Expose the ability to share saved objects to your users*

And then this slightly less specific title could encompass the decision that they have to make:
1) Rely solely on SOM for sharing (probably not our preference?)
2) Introduce a share SO experience within your app
3) A combination of 1 and 2.

----

I know this is a draft, so it's also possible that your planned changes to these docs will make this tip clearer to me as-is.